content/uiux/concepts/button/button.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Button'
+Description: 'A **button** is a design element that allows users to take action.'
+Subjects:
+  - 'Web Design'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'UI'
+  - 'UX'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'intro-to-ui-ux'
+  - 'path/front-end-engineer-career-path'
+---
+
+A **button** is a design element that allows users to take action.
+
+Buttons play an essential role in digital products because they help users complete various actions, such as confirming a purchase, uploading a file, or closing a pop-up modal.
+
+The design of a button should clearly convey its function, so the user understands what will happen if they select it. A button may feature text, an icon, or both to communicate what it does.
+Additionally, [text](https://www.codecademy.com/resources/docs/uiux/text), [images](https://www.codecademy.com/resources/docs/uiux/images), or other design elements near the button can provide context to clarify its purpose.
+
+### Button Elements
+
+- A **text label** describes the button’s function in words. Typically, the text is a verb (“Continue”) or short phrase (“Create Account”).
+- An **icon** is a glyph that symbolizes the button’s function. A button can include an icon and text, or an icon by itself. The icon should adhere to commonly accepted uses so it aligns with user expectations. For example, the thumbs-up icon is widely used to indicate approval or “liking” something.
+- A **container** is a shape that encompasses the button’s text label, icon, or both.
+  - Containers create visual emphasis, allowing the buttons to stand out from other elements. A container with a solid [fill](https://www.codecademy.com/resources/uiux/fill) and is typically easier to spot than an outlined container that has a [stroke](https://www.codecademy.com/resources/uiux/stroke) but no fill.
+  - Buttons without containers have less visual emphasis. Without a container, a text-only button may look like a link. However, a button helps the user complete an action while a link opens a new page or jumps to a specific area on a page.
+
+![Three buttons from Codecademy’s website: 1. Button with a container and a solid fill. 2. Button with an outlined container and no fill. 3. Text-only button with no fill.](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/docs/Buttons-Containers.png)
```

Comment:
I think it'd be a good idea to tie in this picture with the points you have above by referencing parts of the image in each bullet point.

Reply:
For example, for the first bullet point, you can tack on something like, "In the image below..." and some observation about the image below in relation to the point we're making

Reply:
Added.